### PR TITLE
Updates some variable names and match md to guide

### PR DESCRIPTION
**Pull request by m-mayran**
- Renames variables for groups instead of users if related to groups
- Renames groups to match the names in guide
- Merge caip_users and trusted_scientists into one variable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,8 +108,7 @@ module "notebooks_blueprint_security" {
   project_trusted_kms             = "trusted-kms"
   trusted_private_network         = "projects/<shared-restricted-prj>/global/networks/<your_vpc>"
   trusted_private_subnet          = "projects/<shared-restricted-prj>/regions/<region>/subnetworks/<your_subnets_for_notebooks>"
-  caip_users                      = ["trusted1@example.com", "trusted2@example.com"]
-  confid_users                    = ["group:admin@example.com", "group:trusted-users@example.com"]
+  confidential_groups             = ["group:data-owners@example.com", "group:trusted-data-scientists@example.com"]
   trusted_scientists              = ["user:trusted1@example.com", "user:trusted2@example.com"]
 }
 ```
```
Comment (erlanderlo) [inline README.md:114]: In the "inputs" section, also clean up the `caip_users` and `confid_users` terms  (line 169, 171)

**Review by erlanderlo — CHANGES_REQUESTED** [review]
lgtm with minor change needed

File: README.md (hunk 1 above)
Comment (m-mayran) [reply README.md:114]: done

**Review by erlanderlo — APPROVED** [review]
LGTM

**Pull request merged**